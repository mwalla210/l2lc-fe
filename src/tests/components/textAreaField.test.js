import React from 'react'
import renderer from 'react-test-renderer'
import TextAreaField from '../../components/textAreaField'

describe('TextAreaField', () => {
  it ('Renders', () => {
    let options = {
      page: {},
      id: '',
      disabled: false,
      value: '',
      rows: 1,
      onChange: jest.fn(),
      onBlur: jest.fn(),
      focus: false
    }
    const component = renderer.create(
      <TextAreaField {...options}/>,
    )
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
