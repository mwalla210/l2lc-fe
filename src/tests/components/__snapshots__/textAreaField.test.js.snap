// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextAreaField Renders 1`] = `
<textarea
  autoFocus={false}
  className="form-control"
  disabled={false}
  id=""
  onBlur={[MockFunction]}
  onChange={[MockFunction]}
  rows={1}
  style={
    Object {
      "height": "46px",
    }
  }
  value=""
/>
`;
